breacharbiter.go

```diff
@@ -625,6 +645,63 @@ Loop:
 
 			goto justiceTxBroadcast
 
+		// On every new block, we check whether we should republish the
+		// transactions.
+		case <-newBlockChan.Epochs:
+			blocksPassed++
+
+			// If less than four blocks have passed since we
+			// published the justice tx, we'll continue waiting. It
+			// was published with a 2-block fee estimate, so it's
+			// not unexpected that four blocks without confirmation
+			// can pass.
+			if blocksPassed < 4 {
```

Comment:
Ideally should be moved to a constant, or possibly better yet a config value. With this existing value, may be the case that we're somewhat wasteful if rather than malicious activity, its just normal confirmation delay that holds things up. In the end though, I guess we'd rather be safe than sorry, and breaches are still rather rare in practice.

Reply:
Done